apps/modernization-ui/src/apps/search/investigation/GeneralSearchFields.tsx

```diff
@@ -244,17 +245,18 @@ const GeneralSearchFields = () => {
                             rules={{
                                 required: { value: true, message: 'Provider is required' }
                             }}
-                            render={({ field: { name, value, onBlur, onChange }, fieldState: { error } }) => (
-                                <ProviderAutocomplete
-                                    id={name}
-                                    label="Event provider type"
-                                    placeholder=""
-                                    value={value}
-                                    onChange={onChange}
-                                    required={true}
-                                    onBlur={onBlur}
-                                    error={error?.message}
-                                />
+                            render={({ field: { onBlur, onChange, name, value }, fieldState: { error } }) => (
+                                <>
+                                    <ProviderAutocomplete
+                                        id={name}
+                                        label="Event provider type"
+                                        value={value}
+                                        onChange={onChange}
+                                        required={true}
+                                        onBlur={onBlur}
+                                    />
+                                    {error && <ErrorMessage id={`${error}-message`}>{error?.message}</ErrorMessage>}
```

Comment:
question:  Do the auto complete components have the `error` property?

Reply:
yes for auto complete component it does not have the error property, but in the autocomplete the text field does have it so it won't work because of the dropdown selection.

Reply:
Okay Ill look into it